cyw43/src/control.rs

```diff
@@ -16,6 +16,12 @@ pub struct Error {
     pub status: u32,
 }
 
+#[derive(Debug)]
+pub enum AddMulticastAddressError {
+    NotMulticast,
+    NoFreeSlots,
+}
+
```

Comment:
Is this error handling strategy appropriate in this library? I.e. a separate error type for this use-case, not mixed with the other one above. I know it's a matter of convention, let me know what is best here.

Reply:
:+1: